src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -10588,6 +10588,87 @@ public void InsertDeleteMethod_Active()
 
         #endregion
 
+        #region Records
+
+        [Fact]
+        public void Record()
+        {
+            var src1 = @"
+record C(int X)
+{
+    public int X { get; init; } = <AS:0>1</AS:0>;
+}";
+            var src2 = @"
+record C(int X)
+{
+    public int X { get; init; } = <AS:0>2</AS:0>;
+}";
+            var edits = GetTopEdits(src1, src2);
+            var active = GetActiveStatements(src1, src2);
+
+            edits.VerifyRudeDiagnostics(active);
+        }
+
+        [Fact]
+        public void Record_Constructor()
```

Comment:
>Record_Constructor [](start = 20, length = 18)

I don't think this one is needed - it isn't different from the previous one in any significant way.